frontend/public/components/utils/resource-link.tsx

```diff
@@ -90,7 +91,7 @@ export const ResourceLink = connectToModel(
 
     return (
       <span className={classes}>
-        {!hideIcon && <ResourceIcon kind={kind} />}
+        {!hideIcon && <ResourceIcon kind={kind} model={model} />}
```

Comment:
I wonder if there is any risk of the path not having the right icon... since line 86 makes use of the `kind`:
```
 const path = resourcePath(kind, name, namespace);
```